tuichain/api/serializers.py

```diff
@@ -4,19 +4,59 @@
 class StudentSerializer(serializers.ModelSerializer):
     class Meta:
         model = Student
-        fields = '__all__'
+        fields = ('first_name',
+                'last_name',
+                'full_name',
+                'birth_date',
+                'email',
+                'country',
+                'address',
+                'id_number')
+
+    def create(self, validated_data):
+        return Student.objects.create(**validated_data)
+
 
 class InvestorSerializer(serializers.ModelSerializer):
     class Meta:
         model = Investor
-        fields = '__all__'
+        fields = ('first_name',
+                'last_name',
+                'full_name',
+                'birth_date',
+                'email',
+                'country',
+                'address')
+
+    def create(self, validated_data):
+        return Investor.objects.create(**validated_data)
 
 class LoanRequestSerializer(serializers.ModelSerializer):
     class Meta:
         model = LoanRequest
-        fields = '__all__'
+        fields = ('school',
+                'course',
+                'amount',
+                'student')
```

Comment:
why not pass the all the fields here?

Reply:
Guess it's because 'creation_date' is not important

Reply:
But it might be in some cases, not sure

Reply:
and also current amount right?